src/content/docs/api/get-response.mdx

```diff
@@ -31,11 +31,11 @@ import { CodeBracketSquareIcon } from '@heroicons/react/24/outline'
 import { http, HttpResponse, getResponse } from 'msw'
 
 const handlers = [
-  http.get('/user', () => {
+  http.get('*/user', () => {
```

Comment:
This isn't strictly incorrect, as this handler will work both in the browser and in Node.js, given you polyfill `document.baseURI` in Node.js (that happens automatically if you are using environments like JSDOM, and implies that you are about to test code meant for the browser). 

If you think this example would be less ambiguous with an absolute URL, let's make it `https://example.com/user` instead.

Reply:
Thanks for point out,  I don't consider to polyfill it.

fix it
https://github.com/mswjs/mswjs.io/pull/385/commits/ab8d4f9bae3d03ab08a0ab0e1da589aeb57f0bb6